components/src/form-field/error.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from "@angular/core";
+
+import { I18nService } from "jslib-common/abstractions/i18n.service";
+
+// Increments for each instance of the input component
+let nextId = 0;
```

Comment:
I think this is fine, but the comment is misleading. It increments with each instance of this component, not the input component. Same for the pattern in other components.